controller/src/main/java/io/pravega/controller/store/stream/InMemoryStreamMetadataStore.java

```diff
@@ -74,7 +75,7 @@ public InMemoryStreamMetadataStore(Executor executor) {
     }
 
     public Map<String, InMemoryScope> getScopes() {
```

Comment:
please add synchronized here as well as this is reading the `scopes` map which is concurrency protected.

Reply:
removed this method as discussed.